### DISCUSSION TITLE
Vector's CodeLab

### DISCUSSION THREAD

**Discussion by Luc-cos** [cyb3rdog/Cyb3rVector#8] [Q&A]
It could possible using EscapePod.local server with Cyb3rVector extension  (raspberry) to run own script as standalone behaviors ?

**Comment by cyb3rdog** [comment]
Hi, yes, great idea, once, I'll finish the "headless mode" for codelab programs and the "task manager" roadmap milestones, it will be possible to deploy the codelab programs on to the escapepod itself, and therefore even call your own script.

Many thanks for your support and stay tuned!

**Reply by gtchtc07** [reply]
hi I'm new to escape pod and i was hoping that i could use cyb3rvector code lab directly from the PI/EscapePod rather than running it on a separate pc.. is this currently possible or will it be a future possibility  ... or am i somehow able to load the CodeLab save file onto my EscapePod for direct control?

thanks xXx
